_includes/site-navigation/sidenav-mobile.html

```diff
@@ -1,126 +1,133 @@
 {% assign current = page.url | downcase | split: '/' %}
 
 <aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="#">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
+  <button class="usa-button-dismiss sliding-panel-close"> <span class="usa-sr-only">Dismiss</span></button>
   <nav>
```

Comment:
mostly copy+pasted from the code sample in the samwds button documentation, it has the .usa-button-dismiss class added to get the svg/png images as a background image